### PR TITLE
[PUB-714] Add post's stats bar

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
### Purpose
Add post's stats bar to be displayed for Pro Users

### Notes
Task: [PUB-714](https://buffer.atlassian.net/browse/PUB-714)

### CODE REVIEW

File: packages/shared-components/Post/index.jsx [hunk 1]
```diff
@@ -164,6 +168,12 @@ const Post = ({
           dragging={dragging}
           onRequeueClick={onRequeueClick}
         />
+        {sent && isUserPro &&
```
Comment (anafilipadealmeida) [inline packages/shared-components/Post/index.jsx]: Temporary conditions: will be changed once the FeatureLoader and the sent pro fix are up

File: packages/shared-components/QueueItems/story.jsx [hunk 2]
```diff
@@ -9,9 +9,18 @@ import {
   postLists,
   postListsNoHeaders,
 } from './postData';
+import createStore from '@bufferapp/publish-store';
+import { Provider } from 'react-redux';
+
+const store = createStore();
```
Comment (hamstu) [inline packages/shared-components/QueueItems/story.jsx]: I notice you've added the store to all the post types. It's a pretty big chunk of code to bring in to every story. Is there a way we can maybe fake just what we need instead?

File: packages/shared-components/PostStats/index.jsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  
+  return (
+    <div style={statsBarStyle}>
+      {Object.keys(titles).map((typeStats) => createElement(typeStats))}
+    </div>
+  );
+};
+
+PostStats.propTypes = {
+  profileService: PropTypes.oneOf(['twitter', 'linkedin', 'google', 'facebook', 'instagram', 'pinterest']),
```
Comment (hamstu) [inline packages/shared-components/PostStats/index.jsx]: Just a small one, but we have a constants to make this a little shorter and more DRY. :)
https://github.com/bufferapp/buffer-publish/blob/5e62afec097f784d1516fa1a42c722d3c683165a/packages/profile-sidebar/components/ProfileBadge/index.jsx#L35

Comment (anafilipadealmeida) [reply packages/shared-components/PostStats/index.jsx]: Oh, great! I didn't know about that, thanks for showing it to me! 👍

File: packages/shared-components/PostStats/index.jsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  display: 'flex',
+  alignItems: 'baseline',
+  justifyContent: 'center',
+  borderTop: `${borderWidth} solid ${mystic}`,
+  padding: '0.5rem',
+  borderRight: `${borderWidth} solid ${mystic}`,
+};
+
+/* Abbreviates numbers, eg: if we want 2 decimal places, it displays 1240 as 1.24k */
+const abbreviateNumber = (number, decPlaces) => {
```
[the remark this answers is not in the document]
Comment (anafilipadealmeida) [reply packages/shared-components/PostStats/index.jsx]: 💯 Great suggestion, gonna do it!

**Pull request merged**